<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Halo4ProfileDialog</class>
 <widget class="QDialog" name="Halo4ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Halo 4 Profile Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>General Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,0">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Service Tag:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="txtServiceTag">
            <property name="maxLength">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Last in Campaign:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dteCampaignTimestamp"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Experience:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spnExp">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Spartan Points:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spnSpartanPoints">
            <property name="minimum">
             <number>-2147483647</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Campaign Progress</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="4,0,5">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1,1">
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="imgMission">
            <property name="minimumSize">
             <size>
              <width>232</width>
              <height>116</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>232</width>
              <height>116</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/images/images/prologue.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Single Player:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="chkSP">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbxDiffSP">
              <item>
               <property name="text">
                <string>Easy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Normal</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Heroic</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Legendary</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>CoOp:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="chkCoOp">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbxDiffCoOp">
              <item>
               <property name="text">
                <string>Easy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Normal</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Heroic</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Legendary</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1,0,2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Rally Points:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="chkRallyPoints">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbxRallyPoints"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,3,1,1,0">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Max All</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/images/unlock.png</normaloff>:/images/images/unlock.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
